README.md

```diff
@@ -1,53 +1,144 @@
-database.yml defines are database connection and ActiveRecord::Base.connection holds those connections.
+This will get the tests passing, but it still has some weird errors, and this is because we need to create a schema file. You can do that by running ```rake db:migrate```, this will create the schema file and clear the warning. Now update the Post spec to test for a Post being created, it will look something like this:
 
-That's it for models in rails for now.
+```ruby
+describe Post do
+  it 'can be created' do
+    post = Post.create!(title: "My title", description: "The post description")
+    expect(post).to be_valid
+  end
+end
+```
 
-create a migration if needed, for new models for sure
-define the model file, inherit from AR
-use the model.
-lets avoid generators for now....
+Running this test gives us the error of: ```undefined method `create!' for Post:Class```. To implement this feature let's create the database table for our posts. Create a new directory in the ```db/``` directory called ```migrate``` and add a new file called ```001_create_posts.rb``` and add this code:
+
+```ruby
+class CreatePosts < ActiveRecord::Migration
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :description
+
+      t.timestamps null: false
+    end
+  end
+end
+```
+
+This is a basic migration that will create a ```posts``` table that will have title and description columns, along with the built in timestamps. Migrations [documentation](http://edgeguides.rubyonrails.org/active_record_migrations.html). This migration follows the standard naming convention, when you want to create a table, the class name should reflect that, hence the name ```CreatePosts```. This is then reiterated by the argument passed to the ```create_table``` method of ```:posts```. The filename itself needs to be unique, and when you generate a migration automatically through a model or scaffold generator you will notice that the migration file name is prepended with a timestamp value to make sure there are no duplicate migration files.
+
+After running ```rake db:migrate``` we can see that our ```db/schema.rb``` file has been updated with our new posts table, however if we run our tests again you can see that the tests are still failing due to the error: ```undefined method `create!' for Post:Class```. This is because we left out one very important piece of code from the Post model file, in order to leverage built in methods such as ```create!``` we need to have the Post class inherit from ```ActiveRecord::Base```. Update the post.rb model file to match this below:
+
+```ruby
+class Post < ActiveRecord::Base
+end
+```
+
+Now all of the tests are passing and we can create a new post correctly. Even though we know this is working because our tests are passing, let's still test this in the console, open up the Rails console by running: ```rails c```. Once the session has started, run the following command to ensure it recognizes our new Post model:
```

Comment:
`rails console` rather than `rails c` since they may not have used console before. Also might explain more about what's going on with it.